InfiniteLoopBookProject/src/com/st/lms/model/Book.java

```diff
@@ -0,0 +1,38 @@
+package com.st.lms.model;
+
+public class Book extends Entry{
+	private int authorID, publisherID;

```

Comment:
In my opinion, a Book should have *object* references to its author and publisher, not just their IDs. (If we were writing in a language, such as BASIC, with record types but not objects, that would be one thing, but Java gives us the tools to model the relationship directly.)